jenkins/job-configs/kubernetes-jenkins/sen-lu-test-config-upload.yaml

```diff
@@ -0,0 +1,35 @@
+- job:
+    name: 'senlu-test-config-upload'
+    description: 'Build testgrid config from github and upload to gcs'
+    node: 'master'
+    properties:
+        - build-discarder:
+            num-to-keep: 20
+    triggers:
+        - timed: 'H 8 * * 1-5'
```

Comment:
I think you may want to use the `pollscm` trigger like in kubernetes-update-jenkins-jobs.yaml. That will cause the build to run when (but only when) something changes in the test-infra repo.

(though if we go that route, may also want some additional logic to only actually run if something changed in the testgrid config, since most changes won't change the testgrid config.)